patient-service/src/main/java/com/raf/si/patientservice/service/impl/SchedMedExaminationServiceImpl.java

```diff
@@ -221,5 +191,59 @@ private int max(int first, int... rest) {
         }
         return ret;
     }
+
+    private SchedMedExamListResponse schedMedExamPageToSchedMedExamExtendListResponse(Page<ScheduledMedExamination> medExaminationPage) {
+        SchedMedExamListResponse schedMedExamListResponse=schedMedExamMapper.schedMedExamPageToSchedMedExamListResponse
+                (medExaminationPage);
+
+        List<SchedMedExamExtendedResponse> updatedResponses = schedMedExamListResponse.getSchedMedExamResponseList()
+                .stream()
+                .map(schedMedExamResponse -> {
+                    UUID lbp = schedMedExamResponse.getLbp();
+                    PatientResponse patientResponse = patientMapper.patientToPatientResponse(patientRepository.findByLbp(lbp).get());
+                    schedMedExamResponse.setPatientResponse(patientResponse);
+                    return schedMedExamResponse;
```

Comment:
Ovde je problem sto za svaki red u listi mi pozivamo bazu, ne znam zasto u modelu za zakazivanje nemamo pacijenta kao many to one vec imamo samo lbp, time bi resili ovo a i vracali bi svuda pacijenta